src/main/app/src/Components/HomeComponent/Home.tsx

```diff
@@ -0,0 +1,230 @@
+                                 src={process.env.REACT_APP_BACKEND_URL + '/render/' + l.identifier + '/graph.png'}
+                                 style={{maxWidth: 100, float: 'left'}}/>
+                        </Button>
+
+                    </Grid>
+                    <Grid item xs={3}>
+                        <Typography variant="overline" display="block" gutterBottom>
+                            Info
+                        </Typography>
+                        {l.description}
+                        <br/>
+                        <br/>
+                        Identifier: {l.identifier}
+                        <br/>
+                        Contact: {l.contact || '-'}
+                        <br/>
+                        Teams: {l.stats.teams.join(', ')}
+                        <br/>
+
+                    </Grid>
+
+                    <Grid item xs={2}>
+
+                        <Typography variant="overline" display="block" gutterBottom>
+                            State
+                        </Typography>
+                        {l.stats.overallState || '-'}
+
+                    </Grid>
+
+                    <Grid item xs={2}>
+                        <Typography variant="overline" display="block" gutterBottom>
+                            Items
+                        </Typography>
+                        <Typography variant="h2" display="block" gutterBottom>
+                            {l.stats.items}
+                        </Typography>
+                        in {l.stats.groups} groups
+                    </Grid>
+
+                    <Grid item xs={2}>
+                        <Typography variant="overline" display="block" gutterBottom>
+                            Last update
+                        </Typography>
+                        <Typography variant="h3" display="block">
+                            {l.stats.lastUpdate?.split(' ')[0] || '-'}
+                        </Typography>
+
+                        <div>{l.stats.lastUpdate?.split(' ')[1] || '-'}</div>
+                    </Grid>
+
+                    <Grid item xs={2}>
+                        <Typography variant="overline" display="block" gutterBottom>
+                            More
+                        </Typography>
+                        <a target={'_blank'}
+                           rel='noopener noreferrer'
+                           href={process.env.REACT_APP_BACKEND_URL + '/render/' + l.identifier + '/map.svg'}>
+                            Printable Graph
+                        </a><br />
+                        <a target={'_blank'}
+                           rel='noopener noreferrer'
+                           href={process.env.REACT_APP_BACKEND_URL + '/docs/' + l.identifier + '/report.html'}>
+                            Printable Report
+                        </a>
+                    </Grid>
+                </Grid>
+
+            );
+        });
+    }
+
```

Comment:
This is a big component. The landscape could maybe be moved to an own component.

Reply:
we could move it into a layout component (and maybe work with layout components in all components?)